app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -153,6 +153,16 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 				|| teachingAssignment.sabbatical);
 			};
 
+			scope.sortArray = function (array) {
```

Comment:
Rename function. `sortArray` is too generic. What kinds of arrays does it sort, what special operations does it do while sorting, etc.

If it is truly a generic array sort, it probably belongs in a helpers file that gets included in this one.

Reply:
Also, document this function given its specific behavior.